stl/inc/flat_map

```diff
@@ -1257,7 +1282,17 @@ flat_map(from_range_t, _Rng&&, _Compare = _Compare(), _Allocator = _Allocator())
     -> flat_map<_Range_key_type<_Rng>, _Range_mapped_type<_Rng>, _Compare,
         vector<_Range_key_type<_Rng>, _Rebind_alloc_t<_Allocator, _Range_key_type<_Rng>>>,
         vector<_Range_mapped_type<_Rng>, _Rebind_alloc_t<_Allocator, _Range_mapped_type<_Rng>>>>;
-#endif // defined(__cpp_lib_byte)
+#else // ^^^ defined(__cpp_lib_byte) / !defined(__cpp_lib_byte) vvv
+template <_RANGES input_range _Rng, class _Compare = less<_Range_key_type<_Rng>>>
+flat_map(from_range_t, _Rng&&, _Compare = _Compare()) -> flat_map<_Range_key_type<_Rng>, _Range_mapped_type<_Rng>,
+    _Compare, vector<_Range_key_type<_Rng>, allocator<_Range_key_type<_Rng>>>,
+    vector<_Range_mapped_type<_Rng>, allocator<_Range_mapped_type<_Rng>>>>;

```

Comment:
`vector` will default to these `allocator`s, so mentioning them explicitly is unnecessarily verbose.

Occurs below for `flat_multimap`'s deduction guide.